libmscore/image.cpp

```diff
@@ -570,6 +570,8 @@ QVariant Image::propertyDefault(P_ID id) const
 
 void Image::startEdit(EditData& ed)
       {
+      //Image is not editable with drag now, but still initialize ElementEditData for further possible changes
```

Comment:
Sorry I missed that one but images are also editable via drag. Just double click a drag the handle. Please remove the comment.